### PR TITLE
Add deploy to our RPM/DEB S3 repositories

### PR DESCRIPTION
**Pull request by LeonidVas**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -5,8 +5,6 @@ Maintainer: Eugene Blikh <bigbes@tarantool.org>
 Build-Depends: debhelper (>= 9), cdbs,
                cmake (>= 2.8),
                tarantool-dev (>= 1.10.0),
-               libsmall-dev,
-#              libmsgpuck-dev,
```
Comment (Totktonada) [inline debian/control:9]: Built package will depend on whether the libraries are installed within the build container due to [this logic](https://github.com/tarantool/memcached/blob/f5ce080a624134a4e7e05973e3723ef7a405b281/CMakeLists.txt#L37-L55). It should be solved somehow: say, using an option to choose a bundled/external library.

I would choose bundled libraries by default. It is convenient as for a programmer as well as for building our own packages. A downstream distro package build script may use an option to switch to an external library.

File: .github/workflows/packaging.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          CURL_CMD="curl -LfsS \
+            -X PUT ${RWS_URL_PART}/${OS}/${DIST} \
+            -u ${RWS_AUTH} \
+            -F product=${PRODUCT_NAME}"
+
+          for f in $(ls -I '*build*' -I '*.changes' ./build); do
+            CURL_CMD+=" -F $(basename ${f})=@./build/${f}"
+          done
+
+          echo "Upload packages..."
```
Comment (Totktonada) [inline .github/workflows/packaging.yml:77]: If you'll follow my suggestion in https://github.com/tarantool/queue/pull/198#discussion_r1034882541, please, follow it here too.

File: debian/control (hunk 1 above)
Comment (LeonidVas) [reply debian/control:9]: Done.

File: .github/workflows/packaging.yml (hunk 2 above)
Comment (LeonidVas) [reply .github/workflows/packaging.yml]: Done.

File: .github/workflows/packaging.yml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+          PRODUCT_NAME: tarantool-memcached
+        run: |
+          CURL_CMD="curl -LfsS \
+            -X PUT ${RWS_URL_PART}/${OS}/${DIST} \
+            -u ${RWS_AUTH} \
+            -F product=${PRODUCT_NAME}"
+
+        # We don't want to try to print secrets to the log, but we want
+        # to print a "curl" command to see what's going on.
+          CURL_CMD_ECHO="curl -LfsS \
```
Comment (Totktonada) [inline .github/workflows/packaging.yml:75]: Nit: The comment is indented in a surprising way.

**Review by Totktonada — APPROVED** [review]
I didn't test, but visually the patchset looks ok.

File: .github/workflows/packaging.yml (hunk 3 above)
Comment (LeonidVas) [reply .github/workflows/packaging.yml:75]: If I indent `CURL...`, Sublime does not decorates it as a comment. I decided not to argue Sublime.

Comment (LeonidVas) [reply .github/workflows/packaging.yml]: Updated.

**Pull request merged**